### PR TITLE
Lock Nag Worker

### PR DESCRIPTION
**Pull request by russellballestrini**
Add a new Lock Nag Worker to notify users who hold locks for over 3 days.

	new file:   app/messages/lock_nag.rb
	modified:   app/models/environment.rb
	modified:   app/models/lock.rb
	new file:   app/views/messages/lock_nag.text.erb
	new file:   app/workers/lock_nag_worker.rb
	modified:   lib/slashdeploy/service.rb

### CODE REVIEW

File: spec/features/commands_spec.rb [hunk 1]
```diff
@@ -237,6 +237,44 @@
     
   end
 
+  scenario 'david locks an environment and forgets' do
```
Comment (dalyons) [inline spec/features/commands_spec.rb]: hey now

**Comment by dalyons** [comment]
this looks great

**Comment by dalyons** [comment]
id be tempted to make it a day, not 3, given that its a reminder but maybe im a jerk

File: spec/features/commands_spec.rb (hunk 1 above)
Comment (russellballestrini) [reply spec/features/commands_spec.rb]: hah, sorry we use this name a lot in tests.  I think it was the founder originally to be honest.

**Pull request merged**